lib/src/settings.rs

```diff
@@ -105,14 +105,14 @@ impl SignSettings {
 }
 
 fn to_timestamp(value: ConfigValue) -> Result<Timestamp, Box<dyn std::error::Error + Send + Sync>> {
-    // Since toml_edit::Datetime isn't the date-time type used across our code
-    // base, we accept both string and date-time types.
+    fn parse_literal(literal: &str) -> Result<Timestamp, Box<dyn std::error::Error + Send + Sync>> {
+        time_util::parse_datetime(literal).map_err(|err| err.into())
+    }
```

Comment:
nit: inline this as `Ok(time_util::parse_datetime(s)?)`

I think it's also good to extract `time_util::parse_from_rfc3339()` and use it.